int32 status # 1: accepted, 0: rejected 
string sessionID
string UAVID
string messageType

